test/smoke/src/utils.ts

```diff
@@ -130,15 +131,16 @@ export function getRandomUserDataDir(options: ApplicationOptions): string {
 
 export function installCommonAfterHandlers(opts: minimist.ParsedArgs, appFn?: () => Application | undefined, joinFn?: () => Promise<unknown>) {
 	afterEach(async function () {
-		const app: Application = appFn?.() ?? this.app;
+		// {{SQL CARBON EDIT}} - No clear way to take a screenshot
```

Comment:
Do we have an issue tracking re-enabling this? Smoke test failures can be tricky to debug without screenshots so having those would be very helpful.

Reply:
Issue is here: https://github.com/microsoft/azuredatastudio/issues/24002